cumulusci/tasks/metadata/tests/test_package.py

```diff
@@ -195,7 +221,9 @@ def test_parser(self):
 
     def test_parser__missing_item_xpath(self):
         with self.assertRaises(ParserConfigurationError):
-            parser = MetadataXmlElementParser("TestMDT", None, "test", False)
+            parser = MetadataXmlElementParser(  # noqa: F841
+                "TestMDT", None, "test", False
+            )
```

Comment:
Ditto.

Reply:
I added a dummy assert that `parser` is not `none` so the variable is used.  I don't know what needs to be tested: test name is `test_parser__missing_item_xpath`

Reply:
There's an implicit assertion that the parser can be instantiated without an error. The test is probably here mostly to ensure coverage of a particular code path during instantiation.